app/models/account_config.rb

```diff
@@ -69,8 +69,8 @@ def account_type_to_route(class_string)
   # and the NullObject always returns `true` for cross_facility?.
   def account_types_for_facility(facility, action)
     types = account_types
-    types = account_types.select { |type| type.constantize.cross_facility? } if facility.try(:cross_facility?)
-    types = account_types - creation_disabled_types if action == :create
+    types = types.select { |type| type.constantize.cross_facility? } if facility.try(:cross_facility?)
```

Comment:
Possibly this is nit-picky, but couldn't you just keep the old line 72 and get rid of line 71? Was there a reason you chose to keep the separate lines?

Reply:
We need 71 because each of the subsequent lines have a conditional, so we need the initial setup. If 72 wasn't conditional, then you would be right. Am I understanding what you're asking correctly?

Reply:
Oh oops, my eyes just slid past the conditional in 72. That makes sense then.